MdeModulePkg/Library/UefiVariablePolicyLib/UefiVariablePolicyLib.inf

```diff
@@ -36,3 +36,8 @@
   BaseMemoryLib
   MemoryAllocationLib
   SafeIntLib
+
+
+[BuildOptions]
+  MSFT:NOOPT_*_*_CC_FLAGS   = -DINTERNAL_UNIT_TEST
```

Comment:
Should these test flags really be in this library?

Reply:
It's only when building in NOOPT.
This is the best way I've found to do certain internal tests so far. I *do* think it's a place for improvement.

Reply:
wouldn't a PCD be better?